Update @cypress peer dependency with new version

Please update the @cypress peer dependency to allow ^13

Currently while installing the latest version of cypress-axe, with cypress 13.0.0, the following error is displayed:

npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: comet-starter@1.0.0
npm WARN Found: cypress@12.17.4
npm WARN node_modules/cypress
npm WARN   dev cypress@"13.0.0" from the root project
npm WARN   1 more (cypress-axe)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer cypress@"^10 || ^11 || ^12" from cypress-axe@1.4.0
npm WARN node_modules/cypress-axe
npm WARN   dev cypress-axe@"1.4.0" from the root project

--- a/.github/workflows/node.js.yml
+++ b/.github/workflows/node.js.yml
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         node-version: [lts/-2, lts/-1]
-        cypress-version: [10, 11, 12]
+        cypress-version: [10, 11, 12, 13]
 
     steps:
       - uses: actions/checkout@v3

--- a/package.json
+++ b/package.json
@@ -30,7 +30,7 @@
   "dependencies": {},
   "peerDependencies": {
     "axe-core": "^3 || ^4",
-    "cypress": "^10 || ^11 || ^12"
+    "cypress": "^10 || ^11 || ^12 || ^13"
   },
   "devDependencies": {
     "@types/node": "^14.14.8",

